apps/aecontract/src/aect_sophia.erl

```diff
@@ -138,7 +137,7 @@ on_chain_call(ContractKey, Function, Argument) ->
                     , gas => 100000000000000000
                     , gasPrice => 1
                     , origin => 0
-                    , value => 0
+                    , value => Amount
```

Comment:
Not sure what is impact of this.

Reply:
None, Amount is 0 defined a few lines above but not used.

Reply:
🤦🏻‍♂️